pkg/vault/test/foundry/unit/ERC20MultiTokenTest.t.sol

```diff
@@ -284,9 +272,7 @@ contract ERC20MultiTokenTest is Test, IERC20Errors, ERC20MultiToken {
         token.manualTransfer(POOL, OWNER, OWNER2, transferAmount);
     }
 
-    // #endregion
-
-    // #region Private functions
+    // Private functions
```

Comment:
Here again, I think this can just be removed. We used to do this sometimes in v2, but in v3 we haven't been labelling groups by visibility. We use big block headers /****. ****/ in code, but I don't think we need those in tests.